source/transactions-convenient-api/tests/transaction-options.yml

```diff
@@ -0,0 +1,258 @@
+            readConcern: ~
+            startTransaction: ~
+          command_name: commitTransaction
+          database_name: admin
+    outcome: *outcome
+  -
+    description: withTransaction inherits transaction options from defaultTransactionOptions
+    sessionOptions:
+      session0:
+        defaultTransactionOptions:
+          readConcern: { level: snapshot }
+          writeConcern: { w: 1 }
+    operations: *operations
+    expectations:
+      -
+        command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - { _id: 1 }
+            ordered: true
+            lsid: session0
+            txnNumber: { $numberLong: "1" }
+            startTransaction: true
+            autocommit: false
+            readConcern: { level: snapshot }
+            # omitted fields
+            writeConcern: ~
+          command_name: insert
+          database_name: *database_name
+      -
+        command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber: { $numberLong: "1" }
+            autocommit: false
+            writeConcern: { w: 1 }
+            # omitted fields
+            readConcern: ~
+            startTransaction: ~
+          command_name: commitTransaction
+          database_name: admin
+    outcome: *outcome
+  -
+    description: withTransaction explicit transaction options
+    operations: &operations_explicit_transactionOptions
+      -
+        name: withTransaction
+        object: session0
+        callback:
+          operations:
+            -
+              name: insertOne
+              object: collection
+              arguments:
+                session: session0
+                document: { _id: 1 }
+              result:
+                insertedId: 1
+        transactionOptions:
+          readConcern: { level: snapshot }
+          writeConcern: { w: 1 }
+    expectations:
+      -
+        command_started_event:
+          command:
+            insert: *collection_name
+            documents:
+              - { _id: 1 }
+            ordered: true
+            lsid: session0
+            txnNumber: { $numberLong: "1" }
+            startTransaction: true
+            autocommit: false
+            readConcern: { level: snapshot }
+            # omitted fields
+            writeConcern: ~
+          command_name: insert
+          database_name: *database_name
+      -
+        command_started_event:
+          command:
+            commitTransaction: 1
+            lsid: session0
+            txnNumber: { $numberLong: "1" }
+            autocommit: false
+            writeConcern: { w: 1 }
+            # omitted fields
+            readConcern: ~
+            startTransaction: ~
+          command_name: commitTransaction
+          database_name: admin
+    outcome: *outcome
+  -
+    description: withTransaction explicit transaction options override client options
```

Comment:
I think you may have mixed up the descriptions for this test and the one that begins on L223

Reply:
Fixed in https://github.com/mongodb/specifications/pull/435/commits/6e91e113d22e512706a05dc3d65d54ec06e1ae6e. Thanks!